frames/src/main/java/com/checkout/frames/component/expirydate/ExpiryDateComponentState.kt

```diff
@@ -6,4 +6,22 @@ internal data class ExpiryDateComponentState(
     val inputState: InputComponentState
 ) {
     val expiryDate = inputState.inputFieldState.text
+
+    fun hideError() = switchErrorVisibility(false)
+
+    fun showError(errorMessageId: Int) = with(inputState.errorState) {
+        textId.value = errorMessageId
+        switchErrorVisibility(true)
+    }
+
+    fun showError(errorMessage: String) = with(inputState.errorState) {
```

Comment:
Please can you add unit test

Reply:
https://github.com/checkout/frames-android/pull/116#discussion_r959295401